src/Tribe/Service_Providers/Context.php

```diff
@@ -259,21 +259,39 @@ static function ( $data ) {
 				],
 				'tec_post_type'        => [
 					'read' => [
+						Tribe__Context::FUNC          => [
+							static function () {
+								$found = [
+									! empty( tribe_get_request_var( TEC::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Venue::POSTTYPE, false ) ),
+									! empty( tribe_get_request_var( Organizer::POSTTYPE, false ) ),
+								];
+								$found = array_filter( $found );
+
+								return count( $found ) ?: Tribe__Context::NOT_FOUND;
+							},
+						],
 						Tribe__Context::LOCATION_FUNC => [
 							'post_type',
 							static function ( $post_type ) {
-								return count(
+								$found = count(
 									array_intersect(
 										(array) $post_type,
-										[ TEC::POSTTYPE, Venue::POSTTYPE, Organizer::POSTTYPE ]
+										[ TEC::POSTTYPE, Venue::POSTTYPE, Organizer::POSTTYPE, ]
```

Comment:
* :no_entry_sign: Comma not allowed after last value in single-line array declaration
* :no_entry_sign: Comma not allowed after last value in single-line array declaration

<sub>hash: 65aaf622924c2792cdf90d01c81b5021</sub>